pytx3/pytx3/TE.py

```diff
@@ -177,6 +177,32 @@ def getInfoForIDs(self, ids, **kwargs):
 
         return descriptors
 
+    # ----------------------------------------------------------------
```

Comment:
The actual `experimentalfetch` command is ~37% of the loc of `fetch`, granted this may change a little bit as I update experimentalfetch once threat_updates is updated.

The code is also much simpler as threat_updates is a much more direct way to download data compared to what `fetch` does.

Reply:
Incredible!!!